cristin-import/src/test/java/cucumber/GeneralMappingRules.java

```diff
@@ -132,6 +137,18 @@ public void theNvaResourceHasACreationDateEqualTo(String expectedIsoInstant) {
         assertThat(scenarioContext.getNvaEntry().getCreatedDate(), is(equalTo(expectedInstant)));
     }
 
+    @Then("the NVA Resource has a Published Date equal to {string}")
+    public void theNvaResourceHasAPublishedDateEqualTo(String expectedIsoInstant) {
+        Instant expectedInstant = Instant.parse(expectedIsoInstant);
+        assertThat(scenarioContext.getNvaEntry().getPublishedDate(), is(equalTo(expectedInstant)));
+    }
+
+    @Then("the NVA Resource has a Modified Date equal to {string}")
+    public void theNvaResourceHasAModifiedDateEqualTo(String expectedIsoInstant) {
+        Instant expectedInstant = Instant.parse(expectedIsoInstant);
+        assertThat(scenarioContext.getNvaEntry().getModifiedDate(), is(equalTo(expectedInstant)));
```

Comment:
Does this mean that we will create an entry in the database where the creation time will be after the modified date?

Reply:
Also, I think that modified date is controlled by the nva-publication-service. it does not matter what we entrer

Reply:
I do believe the Cristin entry comes with a seperate value for when it was created, so we use that one and not the one created by dynamoDB. I was told to map the Cristin value last_modified_date to the Modified date in NVA